tests/services/case-service.test.js

```diff
@@ -158,7 +158,7 @@ describe('Case service', () => {
   })
 
   it('should call the API to request case list data', async () => {
-    moxios.stubRequest(`${apiUrl}/court/SHF/cases?date=2020-01-01`, {
```

Comment:
Hi, just want to check if you are aware that the VERSION2 of that api paginates the results by default it returns 20 results. Is that ok?

Also the related court-case-service ticket is [here](https://dsdmoj.atlassian.net/browse/PIC-2377)

Reply:
Hey thankd @Obsiye , this was discussed with @johnyfarrar and accordingly made changes

Reply:
and yes, thats fine